frontend/src/components/Navbar.jsx

```diff
@@ -1,164 +1,65 @@
+          </Link>
           <div className="hidden md:block">
-            <div className="ml-10 flex items-baseline space-x-4 text-matic">
-              <a
-                href="/"
-                className=" hover:bg-gray-700 hover:text-white  px-3 py-2 rounded-md text-sm font-medium"
-              >
-                Home
-              </a>
-
-              <a
-                href="/"
-                className=" hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-              >
-                Transfer
-              </a>
-
-              <a
-                href="/"
-                className=" hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-              >
-                About us
-              </a>
-
-              <a
-                href="/FAQs"
-                className=" hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-              >
-                FAQS
-              </a>
-
-              <a
-                href="/"
-                className=" hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-              >
-                Support
-              </a>
-
-              <a
-                href="/login"
-                className=" hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-              >
-                Login
-              </a>
+            <div className="ml-10 flex justify-center items-center space-x-4 text-matic">
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><MdOutlineHome className='mr-[1px]' size='1.2em'/> Home</Link>
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><RiBankLine className='mr-[2px]' size='1.2em'/> Transfer</Link>
+              {
+                (user)
+                ?
+                (
+                  <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><BiWalletAlt className='mr-[1px]' size='1.2em'/> Wallet</Link>
+                )
+                :
+                (
+                  ''
+                )
+              }
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><MdOutlinePersonOutline className='mr-[1px]' size='1.2em'/> About Us</Link>
+              <Link to="/faqs" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold">FAQs</Link>
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold">Support</Link>
```

Comment:
Isn't line 41 a link to the support route?

Reply:
That route is yet to be created.

Reply:
So why do you have it in that function to render the footer? 

You can just add the /support here tho👍